drivers/sensor/lsm6dsl/lsm6dsl.h

```diff
@@ -159,24 +159,6 @@
 #define LSM6DSL_MASK_CTRL3_C_SW_RESET			BIT(0)
 #define LSM6DSL_SHIFT_CTRL3_C_SW_RESET			0
 
-#define LSM6DSL_REG_CTRL3_C				0x12
-#define LSM6DSL_MASK_CTRL3_C_BOOT			BIT(7)
-#define LSM6DSL_SHIFT_CTRL3_C_BOOT			7
-#define LSM6DSL_MASK_CTRL3_C_BDU			BIT(6)
-#define LSM6DSL_SHIFT_CTRL3_C_BDU			6
-#define LSM6DSL_MASK_CTRL3_C_H_LACTIVE			BIT(5)
-#define LSM6DSL_SHIFT_CTRL3_C_H_LACTIVE			5
-#define LSM6DSL_MASK_CTRL3_C_PP_OD			BIT(4)
-#define LSM6DSL_SHIFT_CTRL3_C_PP_OD			4
-#define LSM6DSL_MASK_CTRL3_C_SIM			BIT(3)
-#define LSM6DSL_SHIFT_CTRL3_C_SIM			3
-#define LSM6DSL_MASK_CTRL3_C_IF_INC			BIT(2)
-#define LSM6DSL_SHIFT_CTRL3_C_IF_INC			2
-#define LSM6DSL_MASK_CTRL3_C_BLE			BIT(1)
-#define LSM6DSL_SHIFT_CTRL3_C_BLE			1
-#define LSM6DSL_MASK_CTRL3_C_SW_RESET			BIT(0)
-#define LSM6DSL_SHIFT_CTRL3_C_SW_RESET			0
```

Comment:
This doesn't look like correcting typos. Did you mean to remove this section? If so, please mention in the commit message.

Reply:
Thanks, looks good